invenio_records_rest/__init__.py

```diff
@@ -321,12 +321,12 @@
 function implementing it:
 
 >>> from invenio_records_rest.query import default_search_factory
->>> from elasticsearch_dsl.query import Q
+>>> from invenio_search.engine import dsl
```

Comment:
```suggestion
>>> from invenio_search.engine.dsl import Q
```

wouldn't that work?

Reply:
no, it won't work (it's not a module), we can do from `opensearch_dsl import Q`, but tbh since it's an invenio package I would stick with the import style that is applied by invenio